salt/pillar/mongo.py

```diff
@@ -205,7 +206,8 @@ def ext_pillar(minion_id,
                        'is not formed as a list'.format(re_minion_id))
                 log.error(msg)
             else:
-                included_data = {}
+                included_data = copy.deepcopy(result)
```

Comment:
I'm not sure about this line. It works for me (TM), but people who have enabled "pillar_merge_lists" would have duplicated lists (if there are any in the doc!). What do you think about it?